packages/commonwealth/scripts/archive-outbox.ts

```diff
@@ -195,6 +197,7 @@ if (require.main === module) {
   main()
```

Comment:
Can we load the `env` var in this file too - I noticed that I didn't get Rollbar errors which probably means the env var is not initialized. I incorrectly assumed env var were injected automatically but it seems not.